src/content/en/2020/seo.md

```diff
@@ -23,3 +23,803 @@ queries: 07_SEO
+
+{{ figure_markup(
+  image="seo-canonical-implementation-method.png",
+  caption="Canonical Implementation Method",
+  description="Bar chart showing the canonical tag implementation method. We observed that only 1.03% of desktop homepages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML head of a page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=542127514&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sql"
+  )
+}}
+
+When analyzing the canonical tag implemented in the raw HTML vs. those relying on client side JavaScript rendering, it was identified that 0.68% of the mobile home pages and 0.54% of the desktop ones are detected to be featuring a canonical tag in the rendered but not the raw HTML. This means that there’s only a very small number of pages that are relying on JavaScript to implement canonical tags.
+
+On the other hand, in 0.93% of the mobile home pages and 0.76% of the desktop ones it was seen that there are canonical tags implemented via both the raw and the rendered HTML and there’s a conflict between the URL specified in the raw vs. the rendered HTML of the same pages, which can generate issues.
+
+A similar conflict can be found with the different implementation methods, with 0.15% of the mobile home pages and 0.17% of the desktop ones showing conflicts between the canonical tags implemented via their HTTP headers and HTML head.
+
+### Content
+
+The primary purpose that both search engines and Search Engine Optimization serve is to give visibility to content that users need. Search engines extract features from pages to determine what the content is about. In that way the two are symbiotic. The features extracted align with signals that indicate relevance and inform ranking.
+
+To understand what search engines are able to effectively extract, we have broken out the components of that content and examined the incidence rate of those features between the mobile and desktop contexts. We also reviewed the disparity between mobile and desktop content. The mobile vs desktop disparity is especially valuable because Google has moved to Mobile First Indexing (MFI) for all new sites and as of March of 2021 will move to a mobile-only index wherein content that does not appear within the mobile context will not be evaluated for ranking.
+
+#### Text
+
+##### Rendered vs Non-rendered Text Content
+The usage of Single Page Application (SPA) JavaScript technologies has exploded with the growth of the web. This design pattern introduces difficulties for search engine spiders because both the execution of JavaScript transformations at runtime and user interactions with the page after load can cause additional content to appear or be rendered.
```

Comment:
@aleyda can we add introductory text between these headings so we don't have any "empty headings".

Feel free to use as a TODO for now, so we don't hold up merging this as per @rviscomi 's suggestion, if you need some time to think about what to put here.

```suggestion
#### Text

{# TODO add introductory text to avoid empty heading #}

##### Rendered vs Non-rendered Text Content
The usage of Single Page Application (SPA) JavaScript technologies has exploded with the growth of the web. This design pattern introduces difficulties for search engine spiders because both the execution of JavaScript transformations at runtime and user interactions with the page after load can cause additional content to appear or be rendered.
```

Reply:
Hi @bazzadp - I've solved this issues but in another way, eliminating the need of an intro text there avoiding the text vs. image split (I eliminated that split) and leaving "image" at the same level than other text elements. Thanks!

Reply:
Good stuff. The full 6 levels of headings was probably a little much so think that’s the better solution anyway.